.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Triage: Needed"]
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: happened
+    attributes:
+      label: What happened?
+      description: "Tell us what happened. Provide a detailed description of the issue with context."
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: What you expected?
+      description: "Tell us what you expected to happen."
+    validations:
+      required: true
+  - type: textarea
+    id: possible-fix
+    attributes:
+      label: Possible fix?
+      description: "Tell us about changes that resolve or influence the issue."
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to reproduce?
+      description: "Tell us how to reproduce the issue. Walk us through it step by step."
+      value:
+        1.
+        2.
+        3.
+        ...
+    validations:
+      required: true
+  - type: textarea
+    id: debuglog
+    attributes:
+      label: Debug log
+      description: "Share a Kodi debug log paste URL or file that demonstrates the problem."
```

Comment:
since it is possible to insert help descriptions, i would suggest writing something as done on ISAdaptive repo, see "debug log" line:
https://github.com/xbmc/inputstream.adaptive/issues/new?assignees=&labels=Triage%3A+Needed&projects=&template=01_bug_report.yml&title=%5BBug%5D+
where the links are also specified,
it will help many users who don't know what "paste url" means or how to enable the log